.github/workflows/android-check.yaml

```diff
@@ -86,8 +87,6 @@ jobs:
 
       - name: Checkout sources
         uses: actions/checkout@v4
-        with:
-          fetch-depth: 200 # enough to get all commits for the current day
```

Comment:
Removing it will break the current versioning script. It may be safer to increase the number instead of using default 1.